e2etests/tests/bgp_session_status.go

```diff
@@ -408,3 +409,21 @@ func statusFormatFor(ip string) string {
 func ipFor(statusIP string) string {
 	return strings.ReplaceAll(statusIP, "-", ":")
 }
+
+// Ensures that all the frr-k8s daemons run a clean FRR config.
+// TODO: this is needed for a CI flake, remove once we move to FRR 10.x
+func ensureCleaned(cl client.Client) {
```

Comment:
I would name it  something `WaitFRRNodeStatusesToBeCleaned` just not generic `ensureCleaned` that will force the reader to jump there

Reply:
done